_layouts/default.html

```diff
@@ -63,6 +63,54 @@
         <script defer src="/js/bootstrap.min.js"></script>
         <script defer src="/js/scripts.js"></script>
 
+        <!-- Schema.org JSON-LD Markup for Search Engine Optimization -->
+        <script type='application/ld+json'>
+        {
+            "@context": "https://schema.org/",
+            "@type": "SoftwareApplication",
+            "applicationCategory": "FinanceApplication",
+            "name": "Bisq",
+            "image": "https://bisq.network/images/bisq-logo.svg",
+            "availableOnDevice": "Desktop",
+            "author": {
+            "@type": "Project",
+            "url": "https://bisq.network/dao/",
+            "name": "Bisq Decentralized Autonomous Organization",
```

Comment:
```suggestion
            "name": "Bisq",
```

You're technically correct about Bisq being a DAO, but we usually refer to the body of the project as "Bisq" instead of the "Bisq DAO". Best practical example: see copyright on website footer...it's just plain old "Bisq" :)